specification/cognitiveservices/data-plane/CustomVision/Prediction/stable/v3.1/Prediction.json

```
@@ -82,11 +79,8 @@
     },
     "/{projectId}/classify/iterations/{publishedName}/image/nostore": {
       "post": {
-        "tags": [
-          "ImagePredictionApi"
-        ],
-        "summary": "Classify an image without saving the result.",
-        "operationId": "ClassifyImageWithNoStore",
+        "description": "Classify an image without saving the result.",
+        "operationId": "ImageClassification_ImageNonpersistent",
```

Naming is hard.  Suggest using similar terminology as REST API to improve discoverability.
```suggestion
        "operationId": "ImageClassifier_ClassifyImageWithoutStoring",
```